### PR TITLE
Change OAuth2 RS Origin from the CLI

### PR DESCRIPTION
**Pull request by Water261**
Fixes #
#2353 
Checklist

- [x] This pr contains no AI generated code
- [x] cargo fmt has been run
- [x] cargo clippy has been run
- [ ] cargo test has been run and passes
- [ ] book chapter included (if relevant)
- [ ] design document included (if relevant)

### CODE REVIEW

File: tools/cli/src/cli/oauth2.rs [hunk 1]
```diff
@@ -305,6 +306,25 @@ impl Oauth2Opt {
...
+            Oauth2Opt::SetOrigin { nopt, origin } => {
+                let client = nopt.copt.to_client(OpType::Write).await;
+                match client
+                    .idm_oauth2_rs_update(
+                        &nopt.name,
+                        None,
+                        None,
+                        Some(origin),
+                        None,
+                        true,
```
Comment (yaleman) [inline tools/cli/src/cli/oauth2.rs:318]: The bools at the end shouldn't be true, they reset all the secrets associated with the RS.

**Review by yaleman — CHANGES_REQUESTED** [review]

File: tools/cli/src/cli/oauth2.rs (hunk 1 above)
Comment (Water261) [reply tools/cli/src/cli/oauth2.rs:318]: 👍 Was just a guess, didn't actually know what to do

**Review by yaleman — APPROVED** [review]

File: tools/cli/src/cli/oauth2.rs (hunk 1 above)
Comment (yaleman) [reply tools/cli/src/cli/oauth2.rs]: No worries, that's why we have a review process 😄

**Comment by yaleman** [comment]
Thanks!

**Comment by andreas-bulling** [comment]
Hm, I was just looking for this feature but while it looked as if this was merged, the command is not available in the CLI. At least not in version 1.10.2

**Comment by andreas-bulling** [comment]
Also not in the latest 1.11.0-dev...

**Comment by Firstyear** [comment]
@andreas-bulling `kanidm system oauth2 add-redirect-url`. In the future please open a new issue instead of posting in old issues.

**Comment by andreas-bulling** [comment]
I assumed that you prefer not to have a large number of issues, particularly new issues on an "old" topic. Plus, commenting here instead of opening a new one directly puts the comment/question in context and all people involved at the time are still involved.

But that's fine - sorry for that. I'll open new issues in the future then.

**Pull request merged**